client/my-sites/patterns/index.node.tsx

```diff
@@ -101,7 +103,7 @@ function renderReadymadeTemplateDetails( context: RouterContext, next: RouterNex
 
 	context.primary = (
 		<PatternsWrapper>
-			<ReadymadeTemplateDetails id={ parseInt( context.params.id ) } />
+			<ReadymadeTemplateDetailsServer id={ parseInt( context.params.id ) } />
```

Comment:
Considering that `ReadymadeTemplateDetailsServer` and `ReadymadeTemplateDetails` are the exact the same component, I wonder if we can remove `ReadymadeTemplateDetailsServer` and just use `ReadymadeTemplateDetails` here.